values-AWS-4.12.yaml

```diff
@@ -1,2 +1,3 @@
 cloudProvider:
-  storageClass: gp2-csi
\ No newline at end of file
+  storageClass: gp3
+  provisioner: ebs.csi.aws.com
```

Comment:
Since the provisioner line is the same across versions, just move it into values-AWS.yaml file